packages/backend/src/models/SpaceModel.ts

```diff
@@ -351,20 +351,23 @@
         };
     }
 
-    async find(filters: {
-        projectUuid?: string;
-        projectUuids?: string[];
-        spaceUuid?: string;
-        spaceUuids?: string[];
-        slug?: string;
-    }): Promise<Omit<SpaceSummary, 'userAccess'>[]> {
+    async find(
+        filters: {
+            projectUuid?: string;
+            projectUuids?: string[];
+            spaceUuid?: string;
+            spaceUuids?: string[];
+            slug?: string;
+        },
+        { trx = this.database }: { trx?: Knex } = { trx: this.database },
```

Comment:
does the selects need to be inside the transaction?

Reply:
in the context of `createNestedSpaceFromBaseProject` it does

Reply:
ah I see it now